recipes/qdldl-python/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "0.1.3" %}
+
+package:
+  name: qdldl-python
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/q/qdldl/qdldl-{{ version }}.tar.gz
+  sha256: 2e98bfcc43de55225b82f5895f8266e79a305b174490c60bc559c1538f1be920
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pip
+    - numpy
+    - pybind11
+    - scipy
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - {{ pin_compatible('scipy') }}
+
+test:
+  requires:
+    - pytest
+  imports:
+    - qdldl
+  source_files:
+    - tests/
+  commands:
+    - pytest tests
+
+about:
+  home: https://github.com/oxfordcontrol/qdldl-python
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: 'Python interface to the QDLDL free LDL factorization routine for quasi-definite linear systems'
+
+  description: |
+    Python interface to the [QDLDL](https://github.com/oxfordcontrol/qdldl/) free
```

Comment:
Please remove markdown formatting. This is a plaintext field

Reply:
Markdown (or at least links) _do_ work, compare e.g. faiss: [recipe](https://github.com/conda-forge/faiss-split-feedstock/blob/master/recipe/meta.yaml#L229) and [output](https://github.com/conda-forge/faiss-split-feedstock#about-faiss-split)

Reply:
The markdown is also rendered on Anaconda.org: https://anaconda.org/conda-forge/faiss